packages/components/src/local/form-elements/forces/types/props.d.ts

```diff
@@ -0,0 +1,31 @@
+import { SelectOption } from '@serge/custom-types'
+
+export default interface PropTypes {
+  /**
+   * The label to show on the form section
+   */
+  label?: string
+  /**
+   * The name of the input, if no input is selected, label is used instead.
+   * Should match the value of 'populate' that should be updated in state.
+   */
+  name?: string
+  /**
+   * The list of options available to the form section
+   */
+  options: SelectOption[]
+  /**
+   * The value to set the 'checked' parameter to
+   */
+  value: string | string[]
+  /**
+   * Sets the onChange state on a parent
+   */
+  onChange?: (data: any) => void
```

Comment:
could you check what gets passed in data please?  I don't know if it's a `SelectOption` or the index of the selected items.

Reply:
it should be string[] @IanMayo